ios/ci_scripts/ci_post_clone.sh

```
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -ueo pipefail
+
+echo "$SENTRY_PROPERTIES_BASE64" | base64 --decode > /Volumes/workspace/repository/ios/sentry.properties
+if [ $? -ne 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)